hubblestack/daemon.py

```diff
@@ -697,8 +697,19 @@ def refresh_grains(initial=False):
     global __returners__
     global __context__
 
+    # 'POP' is for tracking persistent opts protection
+    if os.environ.get('NOISY_POP_DEBUG'):
+        log.error('POP refreshing grains (id=%d)', id(__opts__))
```

Comment:
Should this be logged to log.debug, rather than to log.error?

Reply:
Yeah, you'd think so. That's why it's disabled without the magic symbols.

I use the pop messages to track events in splunk, ... I can't see it in splunk if it's at debug level